pkg/provider/azure.go

```diff
@@ -374,7 +373,7 @@ func NewCloud(configReader io.Reader, callFromCCM bool) (cloudprovider.Interface
 	if err != nil {
 		return nil, err
 	}
-	az.ipv6DualStackEnabled = utilfeature.DefaultFeatureGate.Enabled(consts.IPv6DualStack)
+	az.ipv6DualStackEnabled = true
```

Comment:
Should we go one step further and deprecate/remove that field from Cloud config?

Reply:
This should be feasible as it doesn't appear to be a user-exposed configuration vector (we always relied upon the k8s feature-gate interface).

@feiskyer @lzhecheng @nilo19 thoughts?

Reply:
yes, it could be removed  (this is not part of cloud config file).